README.md

```diff
@@ -100,7 +101,8 @@ Select 1:
 What is the difference between a fork and a clone?
 
 ```text
-Your answer here.
+A 'Fork' is a method utilized to make a copy of a master file created/maintained by someone else, into our own repository hub.
+A 'Clone' is the action we perform on the 'Forked' file to make a copy of it to our local system.
 ```
```

Comment:
:+1:  A clone is a version of someone else's repository downloaded to your computer. A fork is a copy of someone else's repository, not necessarily downloaded to your computer. You can clone a fork; you can't fork a clone.